Sources/HeroDebugView.swift

```diff
@@ -50,7 +50,7 @@ class HeroDebugView: UIView {
   }
 
   var showOnTop: Bool = false
-  var rotation: CGFloat = CGFloat(M_PI / 6)
+  var rotation: CGFloat = CGFloat.pi / 6
```

Comment:
Can you do me a favor. Change these to π too. Or change every π to CGFloat.pi and remove π.